src/main/java/com/amplitude/Status.java

```diff
@@ -20,6 +20,14 @@
   /** environment error.. E.g. disconnected from network */
   SYSTEM_ERROR;
 
+  protected static Boolean hasInvalidAPIKey(String errorMsg){
```

Comment:
I guess this method could be in `Response` class, since this method doesn't really have anything related to status